litebird_sim/simulations.py

```diff
@@ -612,8 +613,9 @@ def create_observations(
         assert (
             self.start_time is not None
         ), "you must set start_time when creating the Simulation object"
+
         assert isinstance(
-            self.duration_s, float
+            self.duration_s, (float, int)
```

Comment:
I can't tell how many times I wrongly passed an integer to this parameter… Thanks, I fully approve this addition.